generative_data_prep/__main__.py

```diff
@@ -248,18 +258,43 @@ def get_output_dir(cmd, output_path, overwrite_output_path):
     elif args.cmd == "data_prep":
         verify_output_file(output_path, overwrite_output_path)
         if os.path.splitext(output_path)[-1] != ".hdf5":
-            raise ValueError(f"The output path {output_path} does not end with .hdf5")
+            raise ValueError(
+                f"The output path {output_path} does not end with .hdf5")
         output_dir = os.path.dirname(output_path)
 
     return output_dir
 
 
+def get_categories(categories: str):
```

Comment:
I am more leaning towards don't give the flexibility of providing these in two ways, otherwise it may confuse people?